src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int FIGURES_COUNT = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[FIGURES_COUNT];
+        for (int i = 0; i < figures.length / 2; i++) {
```

Comment:
Using `figures.length / 2` directly in the loop condition can be error-prone if the length of the array changes or the logic for the distribution of figures changes. It's better to define a constant or a variable for this value.